### PR TITLE
Improved Social Media

### PR DESCRIPTION
**Pull request by Elgayar**
Twitter API in social media section was taking too much space, I fixed
the bug and made it look much nicer.

### CODE REVIEW

**Comment by Elgayar** [comment]
https://elgayar.github.io/gci16.fossasia.org/
Here is a live link

**Comment by Elgayar** [comment]
@SaptakS Please check this pull request instead.

**Comment by Elgayar** [comment]
@checkbot I wasn't adding a mentor..

**Comment by Elgayar** [comment]
@animeshsinghweb bots tho xD

File: index.html [hunk 1]
```diff
@@ -428,7 +428,7 @@ <h2 class="white-header">FOSSASIA on social media</h2>
             <!-- Stop ignoring -->
             <!-- Start ignoring BootLintBear -->
                 <div class="col-md-3">
-                    <a class="twitter-timeline" data-width="370"
+                    <a class="twitter-timeline" data-width="370" data-height = "500"
```
Comment (animeshsinghweb) [inline index.html:431]: How does this code fix the issue you mentioned in the description?
Moreover, I can't find any such issue.
Can you elaborate this with the use of snapshots (esp. before and after, if possible)?

Comment (Elgayar) [reply index.html:431]: Yup, one sec

Comment (Elgayar) [reply index.html:431]: That's before:


That's after:
image

@animeshsinghweb

Comment (Elgayar) [reply index.html:431]: Silly fix I know but it made an improvement yeah? 💃

**Comment by abishekvashok** [comment]
@Elgayar could you fix travis 💯

**Comment by Elgayar** [comment]
@Abhi2424shek I believe that error isn't from my side, I only edited the index.html
But anyways, any tips on how can it be fixed?

**Comment by ankitrgadiya** [comment]
I think travis is falling because @Elgayar edited index.html and danger stopped the build.

**Comment by Elgayar** [comment]
@ankitrgadiya And...? How can I get this fixed?

**Comment by ankitrgadiya** [comment]
@Elgayar I just got to know maintainers can add safe edit tags so that danger will not be a problem. But since I'm not the maintainer so I cannot.

**Comment by Elgayar** [comment]
So I'll have to wait for a mentor to solve this.. Okayy. Thankss

**Comment by animeshsinghweb** [comment]
Fine I merged. :)

**Comment by Elgayar** [comment]
@animeshsinghweb Thanks bruh 👍

**Pull request merged**